static/js/glossary.js

```diff
@@ -0,0 +1,92 @@
+var GitGlossary = {
+  data: null,
+  currentLang: 'en',
+  popupArea: null,
+  popupContent: null,
+
+  init: function() {
+    $.getJSON(baseURLPrefix + 'js/glossary-data.json')
+      .done((data) => this.onDataLoaded(data));
+  },
+
+  onDataLoaded: function(data) {
+    this.data = data['en']; // TODO: handle other languages somehow
```

Comment:
You probably wanted to use `date[this.currentLang]` here?

As to how to handle other languages, this is what I used for the multilingual search (to support searching through, say, the French pages when looking at a French manual page):

https://github.com/git/git-scm.com/blob/c239214e8cb1b16e099b5873ddb754fe4891bdd7/assets/js/application.js#L247